package.json

```diff
@@ -1,17 +1,18 @@
 {
-  "name": "iohook",
-  "version": "0.1.2",
+  "name": "iohook-prebuild-test",
```

Comment:
I had to put the package on NPM to test out the install script (npm link / npm install git+.... weren't good enough for what I needed to test)